packages/theme/components/layout/root-layout/Theme.tsx

```diff
@@ -130,6 +130,19 @@ export function Theme({pageMap, children}: ThemeProps) {
                 <Header flatDocsDirectories={flatDocsDirectories} siteTitle={siteTitle} pageMap={pageMap} />
               </PRCBox>
               <PageLayout rowGap="none" columnGap="none" padding="none" containerWidth="full">
+                <PageLayout.Pane
+                  width="small"
+                  sticky
+                  offsetHeader={65}
+                  padding="none"
+                  position="start"
+                  hidden={{narrow: true}}
+                  divider="line"
+                >
+                  <aside aria-label={`${activeHeaderLink ? activeHeaderLink.title : siteTitle} sidebar`}>
```

Comment:
`PageLayout.Pane` is not polymorphic, so we need to nest the `aside` region here.